css/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
This is a very bloated file, are all of these rules actually needed? Ideally we only want to include the code that has a functional purpose for what is currently in the project